paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEvent.kt

```diff
@@ -27,14 +27,24 @@ internal sealed class PaymentSheetEvent : AnalyticsEvent {
     }
 
     class LoadSucceeded(
+        paymentSelection: PaymentSelection?,
         duration: Duration?,
         override val isDeferred: Boolean,
         override val linkEnabled: Boolean,
     ) : PaymentSheetEvent() {
         override val eventName: String = "mc_load_succeeded"
         override val additionalParams: Map<String, Any?> = mapOf(
             FIELD_DURATION to duration?.asSeconds,
+            FIELD_SELECTED_LPM to paymentSelection.defaultAnalyticsValue,
         )
+
+        private val PaymentSelection?.defaultAnalyticsValue: String
+            get() = when (this) {
+                is PaymentSelection.GooglePay -> "google_pay"
+                is PaymentSelection.Link -> "link"
+                is PaymentSelection.Saved -> paymentMethod.type?.code ?: "unknown"
```

Comment:
```suggestion
                is PaymentSelection.Saved -> paymentMethod.type?.code ?: "saved"
```

Reply:
We should probably be consistent across iOS here, but unknown means something different to me (we didn't know if it was saved or not).